bridges/otellogr/example_test.go

```diff
@@ -17,6 +18,17 @@ func Example() {
 	// Create an logr.Logger with *otellogr.LogSink and use it in your application.
 	logr.New(otellogr.NewLogSink(
 		"my/pkg/name",
-		otellogr.WithLoggerProvider(provider)),
-	)
+		otellogr.WithLoggerProvider(provider),
+		// Optionally, set the log level severity mapping.
+		otellogr.WithLevelSeverity(func(i int) log.Severity {
```

Comment:
I wonder what the use case is for this. When would folks want to change the severity mapping?

Reply:
The log level in logr is the following:
```
>= 0, higher meaning "less important"
```

Reply:
Should we instead provide better default values?

Reply:
The defaults are currently wrong but even if they would better we would need to provide such option.

From pkg documentation

> This gives developers a chance to indicate arbitrary grades of importance for info logs, without assigning names with semantic meaning such as "warning", "trace", and "debug." Superficially this may feel very similar, but the primary difference is the lack of semantics. Because verbosity is a numerical value, it's safe to assume that an app running with higher verbosity means more (and less important) logs will be generated.

I really feel we need to give the users the possibility to map the numerical value into the severity leval semantics of OTel.

Also, I think we should probably encourage people using other bridges whenever possible (like `otelslog`, `otelzap`, `otellogrus`) as these are less abstract and should provide better translations for people that are already using these libraries as "backend" for `logr`.